src/accountsdb/db.zig

```diff
@@ -1313,6 +1362,8 @@ pub const AccountsDB = struct {
     /// as the data field ([]u8) for each account.
     /// Returns the unclean file id.
     pub fn flushSlot(self: *Self, slot: Slot) !FileId {
+        var timer = std.time.Timer.start() catch @panic("Timer unsupported");
```

Comment:
should use `sig.time.Timer` here with a `try` ... - maybe we can move the `catch @panic` into the sig.time.Timer struct to remove the try in another pr